PlayerUI/PlayerUICustomization/objc/PlayerUICustomization/ViewController.m

```diff
@@ -207,6 +207,27 @@ - (IBAction)setNextLayout:(id)sender
             newControlLayout = [self simpleCustomLayout];
             self.layoutLabel.text = @"Simple Custom Controls";
             self.playerView.controlsView.layout = newControlLayout;
+            
+            // Customize the font for the play/pause button
+            // This font is registered in Info.plist
+            UIFont *fontello = [UIFont fontWithName:@"fontello" size:22];
+            BCOVPUIButton *playbackButton = self.playerView.controlsView.playbackButton;
+            playbackButton.titleLabel.font = fontello;
+            playbackButton.primaryTitle = @"\ue801";
+            playbackButton.secondaryTitle = @"\ue802";
+            [playbackButton showPrimaryTitle:YES];
+            
+            // Alternatively you can customize a single-state button
+            // with an image instead
+            /*
+            UIImage *iconImage = [UIImage imageNamed:@"ClosedCaptionIcon"];
```

Comment:
This might be cool except that there are no captions in the video so there's nothing to see. While I realize seeing the button is not the important point here, it might be nice if uncommenting this new sample code provided some kind of visual feedback.